src/components/EmptyPlaceholder/EmptyPlaceholder.tsx

```diff
@@ -1,28 +1,35 @@
-import { Grid, Typography } from '@mui/material'
+import { Button, Grid, Typography } from '@mui/material'
 import empty from '@static/svg/empty.svg'
```

Comment:
when you import icons, first we move them to the icons file to have them in one object 
![image](https://github.com/user-attachments/assets/904496ea-e95e-4c09-ab1c-0118245ceb06)